code/modules/projectiles/gun.dm

```diff
@@ -845,3 +846,9 @@
 		H.using_scope = src
 	else
 		H.using_scope = null
+
+obj/item/gun/wield(user)
```

Comment:
add a / to the very start of the proc , inconsistent with all the other overrides in the folder